Artesian/Query/VersionedQuery.py

```diff
@@ -214,7 +214,7 @@ def forVersion(self, version: str) -> VersionedQuery:
         """ Gets the specified version of a versioned timeseries.
 
             Args:
-                version: int of a specific version. Ex.: forVersion("2021-03-12T14:30:00")
```

Comment:
Comparing with the existing code, this was a previous mistake I take responsability for.

Reply:
Line 146 [`versionLimit: int from which the most updated version should be taken Ex.: versionLimit("2021-03-12T14:30:00"`)
] should be changed from _int_ to **string** as well, and the parameter "versionLimit" should be a bit more in depth explained, in my opinion.